server/src/main/java/org/elasticsearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -432,7 +432,7 @@ Table buildTable(RestRequest request, Index[] indices, ClusterHealthResponse res
 
             table.addCell(indexMetaData.getCreationDate());
             ZonedDateTime creationTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(indexMetaData.getCreationDate()), ZoneOffset.UTC);
-            table.addCell(DateFormatters.forPattern("strict_date_time").format(creationTime));
+            table.addCell(DateFormatter.forPattern("strict_date_time").format(creationTime));
```

Comment:
Does it make sense to hoist creation of the date formatter at least out of the loop instead of recreating it for every row? If date formatters are thread-safe this could even be a class constant?